angular.json

```diff
@@ -20,22 +20,22 @@
             "tsConfig": "projects/storefrontapp/tsconfig.app.json",
             "assets": [
               "projects/storefrontapp/src/favicon.ico",
-              "projects/storefrontapp/src/assets"
+              "projects/storefrontapp/src/assets",
+              "projects/storefrontapp/src/manifest.json"
             ],
             "styles": ["projects/storefrontapp/src/styles.scss"],
             "stylePreprocessorOptions": {
               "includePaths": ["./projects", "./projects/storefrontstyles/scss"]
             },
+            "ngswConfigPath": "projects/storefrontlib/src/lib/pwa/ngsw-config.json",
```

Comment:
@LTiger14 this is not the only thing that needs to be added to the config to make pwa work. You can also find things like `serviceWorker: true` that are responsible for enabling SW bundling. I think we can't totally omit things like this and the ideal solution would be a post/preinstall script for our npm package that will update the `angular.json`

Reply:
I did see the other requirement. The thing is at least for the time being we provide our packages (styles and storefront) so that users can use them with a fresh angular project they generate themselves. We would need to tell user to add these manually because we have no way of influencing their angular.json file. For the development, these should stay here.

Reply:
Afaik we can influence it. What stops you from doing this in a postinstall script?